discovery-provider/src/queries/queries.py

```diff
@@ -2251,3 +2251,20 @@ def get_previously_private_playlist():
         playlist_ids = [result[0] for result in previously_private_results]
 
     return api_helpers.success_response({ 'ids': playlist_ids })
+
+# Get the users for a given creator node ID
+@bp.route("/creator_node/users/<int:sp_node_id>", methods=("GET",))
```

Comment:
@SidSethi - how do you feel about this naming? weird or ok?

Reply:
it's really retrieving users by spID? so `/users/creator_node/<int:sp_node_id>` maybe better?